_posts/patterns/2015-03-25-headings.md

```diff
@@ -3,7 +3,7 @@ layout:         pattern
 title:          Headings
```

Comment:
This was a little trickier than I expected because posts have to be specified using the date format. Fortunately I discovered that Jekyll has another notion called 'collections' which seems to be exactly what we want: https://jekyllrb.com/docs/collections/. I've created two collections: components and design elements.